mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/package-info.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *    Copyright (C) 2015 Apache Software Foundation (ASF)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+/**
+ * This package provides the code necessary to create and interact with an {@link rx.Observable} events
+ * coming for Apache Mesos.
```

Comment:
"...and interact with ~~an~~ {@link rx.Observable} events coming ~~for~~ from Apache Mesos." ?